src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/controller/PartnershipController.java

```diff
@@ -47,6 +47,7 @@ public ResponseEntity<Object> createPartnership(
         var transactionId = transaction.getId();
         var logMap = new HashMap<String, Object>();
         logMap.put(URL_PARAM_TRANSACTION_ID, transactionId);
+        
```

Comment:
There are a number of places in this PR where additional empty lines are being added. Please can they be removed. We should only ever add/have one blank line in order to improve readability.

Minor, but because it's occurred in a number of places, better to mention now.

Reply:
@mwestacott could you be more specific as you have just indicated the one just now and the ones you identified yesterday have been resolved.

Reply:
Added a new comment showing where.